nemo/collections/tts/torch/data.py

```diff
@@ -646,3 +657,148 @@ def _collate_fn(self, batch):
 
         joined_data = self.join_data(data_dict)
         return joined_data
+
+
+class VocoderDataset(Dataset):
```

Comment:
Is there no way to make a common subclass for TTSDataset and VocoderDataset?

Reply:
There is no chance to create common subclass, but I will try to unify code as mush as possible.